fs/dafsclient/rmtclient.cpp

```diff
@@ -340,6 +340,31 @@ IDAFS_Exception *createDafsExceptionV(int code, const char *format, ...)
     return ret;
 }
 
+unsigned getPreferredDafsClientPort(bool external)
+{
+    if (isContainerized())
```

Comment:
Does this mean this is not called when K8s calls bare-metal?  Will this trigger failures?

Reply:
that was a late addition and a mistake, there is 1 context it is called in which is containerized only (which is why added, thinking it was the only place), but it is also refactored code that could be called in general/in bare-metal too.
I'll remove.